dask/tests/test_order.py

```diff
@@ -168,11 +175,10 @@ def test_avoid_upwards_branching_complex(abcde):
     }
 
     o = order(dsk)
-
     assert o[(c, 1)] < o[(b, 1)]
 
 
-@pytest.mark.xfail(reason="this case is ambiguous", strict=False)
+# @pytest.mark.xfail(reason="this case is ambiguous", strict=False)  # this now passes
```

Comment:
Can probably remove this comment.

Can you update the docstring to reflect the behavior? I see that we run `e` before `d`. Can we say that `b` runs before both `b` and `c`? Or does `b` run first?